BitwardenShared/UI/Auth/ProfileSwitcher/ProfileSwitcherItem.swift

```diff
@@ -10,6 +10,7 @@ struct ProfileSwitcherItem: Equatable, Hashable {
         ProfileSwitcherItem(
             color: Color(asset: Asset.Colors.primaryBitwardenLight).opacity(0.12),
             email: "",
+            isLoggedOut: false,
```

Comment:
do we need to update `ProfileSwitcherRowTests` for this new property?

Reply:
Ah looks like this isn't used, so I've removed it.